src/data/generate_registry_data.py

```diff
@@ -1783,16 +1783,7 @@ def gen_registry(registry_file, dycore, outdir, indent,
                                     logger, schema_path=schema_dir,
                                     error_on_noxmllint=error_on_no_validate)
     except CCPPError as ccpperr:
-        cemsg = f"{ccpperr}".split('\n', maxsplit=1)[0]
-        if cemsg[0:12] == 'Execution of':
-            xstart = cemsg.find("'")
-            if xstart >= 0:
-                xend = cemsg[xstart + 1:].find("'") + xstart + 1
-                emsg += '\n' + cemsg[xstart + 1:xend]
-            # end if (else, just keep original message)
-        elif cemsg[0:18] == 'validate_xml_file:':
-            emsg += "\n" + cemsg
-        # end if
+        emsg += f"\n{ccpperr}"
```

Comment:
Should we add a unit test that makes sure the xmllint error message is raised and written out as expected?

Reply:
good call! test added!